uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/LeaveRequestCalendarFeedConfig.tpl

```diff
@@ -23,7 +23,10 @@
             <th>{ts}Actions{/ts}</th>
             </thead>
             {foreach from=$rows item=row}
-              <tr id="LeaveRequestCalendarFeedConfig-{$row.id}" class="crm-entity {$row.class}{if NOT $row.is_active} disabled{/if}" data-hash="{$row.hash}">
+              <tr id="LeaveRequestCalendarFeedConfig-{$row.id}"
+                class="crm-entity {$row.class}{if NOT $row.is_active} disabled{/if}"
+                data-title="{$row.title}"
```

Comment:
Do you think we might need to use `{$row.title|escape}` instead of just `{$row.title}` ? I have the suspicion that this would help in case the user types something like `Calendar Feed "Users taking leave"` (weird, but could happen).

Reply:
Woah, it is not escaped by default? Let me test that.

Reply:
Bug caught! Done!

Reply:
![image](https://user-images.githubusercontent.com/3973243/42461521-8a5b7c9c-8398-11e8-85eb-26f957b7aed4.png)